logistics/scripts/modules/tools/ToolsHelper.psm1

```diff
@@ -243,7 +243,7 @@ function Invoke-DbDeploy {
         [AllowEmptyCollection()]
         $Features = @(),
 
-        [string] $ToolsPath = (Get-ToolsPath)
```

Comment:
why change this? doesn't `Get-ToolsPath` do the same check as below?

Reply:
Either case for initdev and deployment , I will have $env:toolsPath value and local variable $ToolsPath will be empty . so I am removing that default value assignment here .

Reply:
$env:toolsPath = $toolsPath = (Join-Path (Get-RepositoryRoot $implementationRepo) 'tools')  this line is exists in InitializeDevelopmentEnvironment.psm1

Reply:
$env:toolsPath = (Join-Path (Get-RootPath) 'tools') for deployment.psm1   https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/pull/266/files#diff-2774a29b696c69083c03917a3ee108d0cee3ae198c6526db3444b15be11421dcR81